endpoints/_includes/functions.php

```diff
@@ -689,7 +689,7 @@ function voice($current_language = null)
     if (has_setting($current_language . "_voice")) {
         return setting($current_language . "_voice");
     } else {
-        return setting('voice');
+        return setting('alice');
```

Comment:
There is already a default https://github.com/bmlt-enabled/yap/blob/master/endpoints/_includes/functions.php#L73

Reply:
Shut the front door, this makes sense now

Reply:
removed as unneeded